templates/default/delos.css

```diff
@@ -18920,4 +18920,3 @@ table.mceToolbar td {
   border: 1px solid #ddd;
   border-radius: .25em;
 }
-/*# sourceMappingURL=delos.css.map */
```

Comment:
@alex40724 Maybe a bit fussy, but does this really belong to this PR ;)?

Reply:
I will take care of this, before merging.